### ISSUE TITLE
Recon name match & edit distance facets are stale

### ISSUE THREAD

**Issue by tfmorris** [OpenRefine/OpenRefine#991]
The data for these facets is computed at reconciliation time, so if the cell value changes later, they are no longer accurate.  In particular, if you cell value has trailing whitespace, it will reconcile correctly and the names will match modulo the whitespace, but the name match facet will return all false.

We can easily compute this on the fly, so I'm not sure why we need the current approach.

The facet expression for name match is : cell.recon.best.name==value

**Comment by tfmorris** [comment]
Oops.  This was a stale issue queued up months ago that just go synced.  It's low priority since Freebase reconciliation is gone, but I'll leave it open for now in case it's useful for other reconciliation services.

**Comment by wetneb** [comment]
I agree we could remove these hard-coded fields: they could be computed on the fly. It would likely help us reduce the memory footprint of recon objects and stop worrying about these results getting stale.
